core/io/file_access.h

```diff
@@ -152,6 +171,12 @@ class FileAccess : public RefCounted {
 	static Ref<FileAccess> create(AccessType p_access); /// Create a file access (for the current platform) this is the only portable way of accessing files.
 	static Ref<FileAccess> create_for_path(const String &p_path);
 	static Ref<FileAccess> open(const String &p_path, int p_mode_flags, Error *r_error = nullptr); /// Create a file access (for the current platform) this is the only portable way of accessing files.
+
+	static Ref<FileAccess> open_bind(const String &p_path, int p_mode_flags);
```

Comment:
The bound method should use the enum.

```suggestion
	static Ref<FileAccess> open_bind(const String &p_path, ModeFlags p_mode_flags);
```